src/test/java/org/openrewrite/java/logging/ParameterizedLoggingTest.java

```diff
@@ -1080,4 +1080,180 @@ public void run() {
             )
         );
     }
+
+    @Issue("https://github.com/openrewrite/rewrite-logging-frameworks/issues/149")
+    @Test
+    void stringFormatToParameterized() {
+        rewriteRun(
+          spec -> spec.recipe(new ParameterizedLogging("org.slf4j.Logger debug(..)", false)),
+          //language=java
+          java(
+            """
+              import org.slf4j.Logger;
+
+              class Test {
+                  static void method(Logger logger, String annotation, String subtype) {
+                      logger.debug(String.format("Could not parse the FileAnnotation %s into %s", annotation, subtype));
+                  }
+              }
+              """,
+            """
+              import org.slf4j.Logger;
+
+              class Test {
+                  static void method(Logger logger, String annotation, String subtype) {
+                      logger.debug("Could not parse the FileAnnotation {} into {}", annotation, subtype);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Issue("https://github.com/openrewrite/rewrite-logging-frameworks/issues/149")
+    @Test
+    void stringFormatWithMultipleFormats() {
+        rewriteRun(
+          spec -> spec.recipe(new ParameterizedLogging("org.slf4j.Logger info(..)", false)),
+          //language=java
+          java(
+            """
+              import org.slf4j.Logger;
+
+              class Test {
+                  static void method(Logger logger, String name, int count, double percentage) {
+                      logger.info(String.format("User %s has %d items (%.2f%% of total)", name, count, percentage));
+                  }
+              }
+              """,
+            """
+              import org.slf4j.Logger;
+
+              class Test {
+                  static void method(Logger logger, String name, int count, double percentage) {
+                      logger.info("User {} has {} items ({}% of total)", name, count, percentage);
```

Comment:
Not sure about this change yet; perhaps best we do not make changes when there's formatting instructions. 🤔